docs/how_to_config.md

```diff
@@ -57,22 +57,63 @@ bit more "Pythonic" and easier to read.)
 This is a template for `.pre-commit-config.yaml`:
 
 ```yaml
+# Option 1: with native tool
 - repo: https://github.com/jsh9/pydoclint
   rev: <latest_tag>
   hooks:
     - id: pydoclint
       args:
-        - [--style=google, --check-return-types=False]
+        - --style=google
+        - --check-return-types=False
+
+# Option 2: with flake8 plug-in
+- repo: https://github.com/pycqa/flake8
+  rev: <latest_tag>
+  hooks:
+    - id: flake8
+      additional_dependencies:
+        - pydoclint==<latest_tag>
+      args:
+        - --style=google
+        - --check-return-types=False
 ```
 
 If you've already specified all the configuration options in a config file,
 here is how to cite them in `.pre-commit-config.yaml`:
 
 ```yaml
+# Option 1: with native tool
 - repo: https://github.com/jsh9/pydoclint
   rev: <latest_tag>
   hooks:
     - id: pydoclint
       args:
         - "--config=pyproject.toml"
+
+# Option 2: with flake8 plug-in
```

Comment:
Let's break up these 2 options into 2 independent code blocks. Otherwise people may just copy the whole block without reading the comments.

Reply:
Good point haha